examples/full/variables.tf

```diff
@@ -15,36 +15,36 @@
  */
 
 variable "region" {
-  description = "The region the API gateway is deployed in."
+  description = "The region to deploy the ParallelCluster API."
   type        = string
   default     = "us-east-1"
 }
 
 variable "profile" {
   type        = string
-  description = "The aws profile used to deploy parallel clusters."
+  description = "The AWS profile used to deploy resources."
```

Comment:
The description is different here because:
1. examples/clusters uses the profile to deploy only the clusters
2. examples/full uses the profile to deploy PCAPI, networking infra and clusters, so I preferred to keep a generic "resources".